dns-controller/pkg/watchers/servicecontroller.go

```diff
@@ -171,7 +184,7 @@ func (c *ServiceController) updateServiceRecords(service *v1.Service) {
 			}
 		}
 	} else {
-		glog.V(4).Infof("Service %q did not have %s annotation", service.Name, AnnotationNameDnsInternal)
+		glog.V(4).Infof("Service %s/%s did not have %s annotation", service.Namespace, service.Name, AnnotationNameDnsExternal)
```

Comment:
Same as above. 

Are we kosher without throwing an `error` upstream? And is `V(4)` still an okay level to keep this at?

Just smells like a failure is all - so due diligence dictates I ask.

Cheers